### PR TITLE
Nearest loc

### LINKED ISSUE #14: Implement finding nearest key location.
Somewhat dependent on @kangp3 's SQLite stuff.

### LINKED ISSUE #15: Add nearest key location to @response JSON
Dependent on #14

### PR DESCRIPTION
**Pull request by kangp3**
Closes #14 and closes #15

### CODE REVIEW

File: app/src/main/java/xyz/whereuat/whereuat/KeyLocationUtils.java [hunk 1]
```diff
@@ -21,6 +66,36 @@ public static boolean locIsValid(Location loc) {
         return loc != null;
     }
 
+    public static KeyLocation findNearestLoc(Cursor c, Location origin) throws
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/KeyLocationUtils.java]: Can you add a comment here about `c` being a cursor to a query over key locations from the db?

File: app/src/main/java/xyz/whereuat/whereuat/MainActivity.java [hunk 2]
```diff
@@ -148,36 +149,47 @@ public void onScrollStateChanged(AbsListView view, int scroll_state) { }
...
+                    KeyLocationUtils.KeyLocation key_loc = KeyLocationUtils.findNearestLoc(c, loc);
+                    try {
+                        (new HttpRequestHandler(context)).postAtResponse(
+                                (new PreferenceController(context)).getClientPhoneNumber(),
+                                intent.getStringExtra(Constants.TO_PHONE_EXTRA), loc.getLatitude(),
+                                loc.getLongitude(), key_loc,
+                                new Response.Listener<String>() {
+                                    @Override
+                                    public void onResponse(String response) {
+                                        String text = "Got a 200 from the @response POST!";
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: I'd be OK with sneaking a `Log.d` into this PR right here lol

File: app/src/main/java/xyz/whereuat/whereuat/MainActivity.java [hunk 3]
```diff
@@ -171,8 +171,7 @@ public void onPostExecute(Object result) {
                                 new Response.Listener<String>() {
                                     @Override
                                     public void onResponse(String response) {
-                                        String text = "Got a 200 from the @response POST!";
-                                        Toast.makeText(context, text, Toast.LENGTH_SHORT).show();
+                                        Log.d(TAG, "200 from @respones POST");
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: s/respones/response lol

Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: Not sure what you're referring to ;)

**Comment by JuliusAlexanderIV** [comment]
lgtm

**Pull request merged**